p11-kit/Makefile.am

```diff
@@ -93,13 +93,23 @@ install-exec-hook:
 	done
 	$(MKDIR_P) $(DESTDIR)$(p11_package_config_modules)
 
+install-data-hook:
+	$(MKDIR_P) $(DESTDIR)$(systemduserdir)/sockets.target.wants
+	$(LN_S) -f ../p11-kit-remote.socket $(DESTDIR)$(systemduserdir)/sockets.target.wants/p11-kit-remote.socket
+
 uninstall-local:
 	for i in so dylib; do \
 		rm -f $(DESTDIR)$(libdir)/p11-kit-proxy.$$i; \
 	done
+	rm -f $(DESTDIR)$(systemduserdir)/sockets.target.wants/p11-kit-remote.socket
 
 endif
 
+systemduserdir = $(prefix)/lib/systemd/user
+systemduser_DATA = \
+	p11-kit/p11-kit-remote.socket \
```

Comment:
I think this file (p11-kit/p11-kit-remote.socket) should be added to EXTRA_DIST, while p11-kit-remote\@.service.in is automatically added to the distribution as it is listed as AC_CONFIG_FILES.